src/api/layers/layers.api.js

```diff
@@ -63,9 +62,6 @@ const generateClassForLayerConfig = (layerConfig, id, allOtherLayers, lang) => {
             attributions.push(new LayerAttribution(attributionName, attributionUrl))
         }
         switch (type.toLowerCase()) {
-            case 'vector':
-                layer = new GeoAdminVectorLayer(id, opacity, layerConfig.styleUrl, attributions)
-                break
```

Comment:
we do not yet receive vector layers through the backend configuration, so the format isn't really set yet
I could keep the case, and output a different error log for this specific type, maybe it would be clearer